source/includes/SystemSettings/system_settings.md

```diff
@@ -0,0 +1,114 @@
+}
+```
+* To create a System Setting, you need to specify below attributes in the request body. If not authenticated or authenticated user does not have sufficient privileges, a `401 Unauthorized` status is returned.
+
+    ### Attributes
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *property* | `String` | the property name for this System Setting, names can be up to 255 chars, must be unique, and follow a convention of module ID followed by category & name lower camelCase separated by periods for e.g., addresshierarchy.allowFreetext
+    *description* | `String` | a description for the usage of this property
+    *datatypeClassname* | `String` | Data type for this System Setting.OpenMRS provides Custom data type resource, which gives the flexibility to select the data type accordingly             *datatypeConfig* | `String` | An optional identifier from the fulfiller (e.g., lab            *preferredHandlerClassname*  | `String` | Handler subresource for the Custom Data Type used. Can optionally define a specific handler class wants to use (otherwise, the framework will choose the best handler for the chosen DataType)
+    *handlerConfig*  | `String` | Allow the handler have any name and config it wants/needs. This will help to identify the data type unambiguously which has been contained and will allow introspecting
+    *value* | `String` | the value assigned to this system setting 
+    
+## Update a System Setting
+
+
+```console
+POST /systemsetting/:target_systemsetting_uuid
+{
+  "property": "property name",
+  "description": "dummy description",
+  "datatypeClassname": "org.openmrs.customdatatype.datatype.LongFreeTextDatatype",
+  "datatypeConfig": "default",
+  "preferredHandlerClassname":"org.openmrs.web.attribute.handler.LongFreeTextTextareaHandler"
+  "handlerConfig": "default",
+  "value": "dummy value"
+}
+```
+* Update a System Setting with given UUID, this method only modifies properties in the request. If the user not logged in to perform this action, a `401 Unauthorized` status returned.
+
+### Attributes
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *property* | `String` | the property name for this System Setting, names can be up to 255 chars, must be unique, and follow a convention of module ID followed by category & name lower camelCase separated by periods for e.g., addresshierarchy.allowFreetext
+    *description* | `String` | a description for the usage of this property
+    *datatypeClassname* | `String` | Data type for this System Setting.OpenMRS provides Custom data type resource, which gives flexibility to select the data type accordingly             *datatypeConfig* | `String` | An optional identifier from the fulfiller (e.g., lab            *preferredHandlerClassname*  | `String` | Handler subresource for the Custom Data Type used. Can optionally define a specific handler class wants to use (otherwise, the framework will choose the best handler for the chosen DataType)
```

Comment:
I can see some kind of formatting issue here.

Reply:
@Akayeshmantha  I also thought so but the view is fine when I build it should I still refactor it??